web_scraping/gog_scraper/scrape_gog_game.py

```diff
@@ -0,0 +1,102 @@
+"""This script is for extracting all of the details of a single game on GOG."""
+
+from datetime import datetime
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+
+
+def get_html(url: str) -> str:
+    """Return the html file of a given url."""
+
+    page = urlopen(url)
+    html_bytes = page.read()
+    html = html_bytes.decode("utf_8")
+    return html
+
+
+def format_os(os_string: str) -> str:
+    """Turns the OS into the proper format."""
+
+    os_map = {
+        "windows": "Windows",
+        "mac": "Mac",
```

Comment:
Is this not just calling `str.title()`?

Reply:
What do you mean

Reply:
This dictionary is mapping platform names to capitalised platform names.

`'windows'.title() # Windows`

Reply:
Oh yeah you're right.